app/src/main/res/values/strings.xml

```diff
@@ -1405,7 +1405,7 @@
     <string name="donate_your_data_to_science">Donate your data to science</string>
     <string name="open_humans_short">OH</string>
     <string name="you_have_been_signed_out_of_open_humans">You have been signed out of Open Humans</string>
-    <string name="click_here_to_sign_in_again_if_this_wasnt_on_purpose">Click here to sign in a again if this wasn\'t on purpose.</string>
```

Comment:
The blacksplash was needed to escape apostrophe.

Reply:
@rICTx-T1D Sorry. But the "a". Shouldn't be before "again". Correct?

Reply:
Just fixed it. Is it ok now?